Dockerfile

```diff
@@ -0,0 +1,114 @@
+FROM multiarch/alpine:amd64-edge
+MAINTAINER wessel<jimfromsa@gmail.com>
+
+ENV IOP_VERSION  v1.2-Beta
+ENV IOP_PLATFORM linux-64
+
+
+# Ports for node2node, client2node, interprocess communication between local services
+EXPOSE 16980
+EXPOSE 16981
+EXPOSE 16982
+
+
+# Public node id, should be an SHA256 hash of the public key of this node
+#ENV NODEID
+
+# TCP port to serve client (i.e. end user) queries. Optional,default value: 16981
+ENV CLIENTPORT 16981
+
+# TCP port to serve other IoP services running on this node. Optional, default value: 16982
+ENV LOCALPORT 16982
+
+#  TCP port to serve node to node communication. Optional, default value: 16980
+#Should only be locally available
+ENV NODEPORT 16980
+
+ENV IOPLOC_PATH "/data/locnet/"
+
+# Path to config file to load options from. Optional, default value: ~/.iop-locnet/iop-locnet.cfg
+ENV CONFIGFILE "/data/locnet/iop-locnet.cfg"
+
+#Path to db file. Optional, default value: ~/.iop-locnet/locnet.sqlite
+ENV DBPATH  "/data/locnet/iop-locnet.sqlite"
+
+#  Path to log file. Optional, default value: ~/.iop-locnet/debug.log
+ENV LOGPATH  "/data/locnet/debug.log"
+
+#   Externally accessible IP address (ipv4 or v6) to be advertised for other nodes or clients. Required for seeds only, autodetected otherwise.
+#ENV HOST
+#   Internally accessible IP address that is required for mercury services on the same node.
+#ENV LOCALHOST
+
+# Host name of seed node to be used instead of default seeds. You can repeat this option to define multiple custom seed nodes.
```

Comment:
Bad comment, this is not the seed node but your own host id reported to other nodes of the network. Is this my mistake somehow or just some copy-paste problem you did? Also, there's a commented NODEID env var with the right description, one of them probably should be deleted.